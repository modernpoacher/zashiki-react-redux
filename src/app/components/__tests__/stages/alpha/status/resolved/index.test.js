import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import Resolved from '@modernpoacher/zashiki-react-redux/app/components/stages/alpha/status/resolved'

Enzyme.configure({ adapter: new Adapter() })

describe('@modernpoacher/zashiki-react-redux/app/components/stages/alpha/status/resolved', () => {
  it('renders', () => {
    const component = (
      <Resolved
        definitions={[]}
        onChange={jest.fn()}
        onSubmit={jest.fn()}
      />
    )

    expect(shallow(component))
      .toMatchSnapshot()
  })
})
