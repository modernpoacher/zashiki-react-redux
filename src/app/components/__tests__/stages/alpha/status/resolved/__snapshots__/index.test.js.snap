// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/stages/alpha/status/resolved renders 1`] = `
<div
  className="alpha resolved"
>
  <Alpha
    definitions={[]}
    onChange={[MockFunction]}
    onSubmit={[MockFunction]}
  />
  <withRouter(Connect(Component)) />
</div>
`;
